client/src/client.js

```diff
@@ -46,7 +46,12 @@ class Client {
     this.app.configure(restClient.fetch(fetch));
 
     // FIXME: move this into runloop once we figure out how to run it locally during testing
-    this.errorTelemetry.setup();
+    try {
+      this.errorTelemetry.setup();
```

Comment:
Why not, but so you had issues with this I'm curious? I didn't AFAICT, and CI was not barking either.